### PR TITLE
Added SQL command to get current default retention period

### PR DESCRIPTION
**Pull request by nmaludy**
Added SQL a command to the docs for getting the default metric retention period. This is useful for idempotency checks in config management tools like Puppet, Ansible, etc.

### CODE REVIEW

File: docs/metric_deletion_and_retention.md [hunk 1]
```diff
@@ -128,6 +128,12 @@ SQL command to set the default retention policy to two days for all metrics that
 SELECT prom_api.set_default_retention_period(INTERVAL '1 day' * 2)
 ```
 
+SQL command to get the current default retention policy measured in days.
+
+```
+SELECT EXTRACT(day FROM _prom_catalog.get_metric_retention_period('')) AS retention_days;
+```
+
 SQL command to set a custom retention policy for a specific metric.
```
Comment (Harkishen-Singh) [inline docs/metric_deletion_and_retention.md:138]: Can you also add SQL command to get retention period for a specific metric?

```postgresql
SELECT EXTRACT(day FROM _prom_catalog.get_metric_retention_period('<metric_name>'))
```

**Review by Harkishen-Singh — CHANGES_REQUESTED** [review]
Thank you for the PR. Can you also add one for fetching retention period specific to a metric, since we support different individual metric retention periods?

**Comment by nmaludy** [comment]
@VineethReddy02 done

File: docs/metric_deletion_and_retention.md [hunk 2]
```diff
@@ -128,6 +128,18 @@ SQL command to set the default retention policy to two days for all metrics that
...
+SQL command to get the current default retention policy measured in days.
+
+```
+SELECT EXTRACT(day FROM _prom_catalog.get_metric_retention_period('')) AS retention_days;
+```
+
+SQL command to get the current retention policy for the metric `container_cpu_usage_seconds_total` measured in days.
+
+```
+SELECT EXTRACT(day FROM _prom_catalog.get_metric_retention_period(container_cpu_usage_seconds_total')) AS retention_days;
```
Comment (bitwalker) [inline docs/metric_deletion_and_retention.md:140]:
```suggestion
SELECT EXTRACT(day FROM _prom_catalog.get_metric_retention_period('container_cpu_usage_seconds_total')) AS retention_days;
```

**Review by bitwalker — APPROVED** [review]
LGTM! Just one small typo

**Review by bitwalker — APPROVED** [review]

**Review by Harkishen-Singh — APPROVED** [review]

**Pull request merged**